### ISSUE TITLE
full GUI flow + destroy does not work

### ISSUE THREAD

**Issue by sym3tri** [coreos/tectonic-installer#348]
/cc @Quentin-M 

I created a cluster with the GUI, but I am unable to destroy it from the command line.

(SIDE ISSUE: the actual process for doing this destroy is undocumented)

Steps to reproduce:
1. Build GUI off master
2. Run through GUI install
3. Cluster installs successfully
4. Download assets.zip
5. `unzip assets.zip`
6. `cd tectonic*`
7. `export AWS_PROFILE=tectonic`
8. `TERRAFORM_CONFIG=`pwd`/.terraformrc terraform destroy`

Produces output:

```
Do you really want to destroy?
  Terraform will delete all your managed infrastructure.
  There is no undo. Only 'yes' will be accepted to confirm.

  Enter a value: yes

provider.aws.region
  The region where AWS operations will take place. Examples
  are us-east-1, us-west-2, etc.

  Default: us-east-1
  Enter a value: us-west-2

module.ignition-workers.ignition_config.main: Refreshing state... (ID: b1358c789c75d6fe93fe8a283f9a8431ef35a66fee5416dd3a474f3629edf312)
module.bootkube.local_file.kube-ca-key: Refreshing state... (ID: 4d16135d1399d524f2f44e682b611703ec95a6e1)
module.tectonic.template_dir.tectonic: Refreshing state... (ID: 0f289aa49c32c569ec2226212b0752eab03b24d7)
module.tectonic.local_file.tectonic-rkt: Refreshing state... (ID: 304f32e7acfe6859f61e67d45bf0be11c2479f49)
module.tectonic.local_file.tectonic: Refreshing state... (ID: abe70d40995c1d2e63c27e8771f726cfdf3eb01c)
module.bootkube.local_file.kubelet-crt: Refreshing state... (ID: eb21498efc4c38078147572a9d731032cbf755f3)
module.bootkube.local_file.kubelet-key: Refreshing state... (ID: 6832762be42fed81da37d99bfc903727781ed878)
module.bootkube.local_file.kubeconfig: Refreshing state... (ID: 1419815b5c5d178572ea5b466e63a360ff2ec82b)
module.bootkube.local_file.service-account-crt: Refreshing state... (ID: f78990c4faf719363d20d9f97b7bca42099b2c4b)
module.bootkube.local_file.apiserver-key: Refreshing state... (ID: 2c0aa3056df24a9dae49b7358360be34d84b7d67)
module.bootkube.local_file.kube-ca-crt: Refreshing state... (ID: 22dafbcfe0a0877baabcc18a86d4e22126c35432)
module.bootkube.local_file.bootkube: Refreshing state... (ID: 4f69853ede032c1eff0d9eb5f033ef31947a1404)
module.bootkube.local_file.apiserver-crt: Refreshing state... (ID: 19b7d36d825615a6415e6464807c8d3218c19139)
module.bootkube.local_file.service-account-key: Refreshing state... (ID: a440f71d77a08f1dc0ac7bc7a8c9274dbc870280)
module.bootkube.template_dir.bootkube: Refreshing state... (ID: 66e9e209b240258480150202a375546555479e38)
module.etcd.ignition_file.node_hostname.2: Refreshing state... (ID: 20899413024fcb56f9d8fdedfa5ad15fa94e84b31b701d4cc6075d30fae81ef9)
module.etcd.ignition_systemd_unit.etcd3.0: Refreshing state... (ID: 464d099898c614063e0a540d4f9c8fafbc2ab751f5f142695baa329956dfae3a)
module.bootkube.template_dir.bootkube-bootstrap: Refreshing state... (ID: bdfde4c99beae6928b9ca9e6f9c7b98e7161c339)
module.etcd.ignition_systemd_unit.etcd3.1: Refreshing state... (ID: 9d3304f2915fc2259b51a0a5ad12374c58523eab82c376e7895817bf21eaec03)
module.etcd.ignition_config.etcd.1: Refreshing state... (ID: aba5e9991bf82d4169efbbdaa15eadf5356f8e9eff554936d4e8cbca112722de)
module.etcd.ignition_systemd_unit.etcd3.2: Refreshing state... (ID: 4a11bb3c47051d6d6bdf5e9e1d6d1ae16fa5107a682f11033f89e5c8eb876b72)
module.etcd.ignition_file.node_hostname.0: Refreshing state... (ID: 42582826af69a436a5390341256fe057f4cb466a5ef8d44bdea77d71a091ef47)
module.etcd.ignition_config.etcd.2: Refreshing state... (ID: 57e8c7698381a7ff2a9b8d39f3cce6ded4ac56dd0422003d89fb8a1dad395f5d)
module.etcd.ignition_config.etcd.0: Refreshing state... (ID: 7fe8aa0d2d2460f4ae47477d55e25abb948365e3ec12f2016637e2252c271eae)
module.etcd.ignition_file.node_hostname.1: Refreshing state... (ID: 5c91abf93af056100841c461690fdb77dff529667c70a6f0237f33ded6fda3c9)
module.ignition-masters.ignition_config.main: Refreshing state... (ID: 280037081a7b6d2ef05527d19b996c8b262b2f5bfd4fcc0a4069f8dcbbad4f83)
module.bootkube.tls_locally_signed_cert.kubelet: Refreshing state... (ID: 89064944546431467245134143475756616722)
module.bootkube.tls_cert_request.kubelet: Refreshing state... (ID: 74e97b4f336670e707ab2dbbd6869c9287b99258)
module.bootkube.tls_private_key.kubelet: Refreshing state... (ID: 84ed07bb4616f484ee0d2e29f33dffbe07ff897d)
module.tectonic.random_id.kubectl_secret: Refreshing state... (ID: jbnsgeonJN12n5OLxwRwcw)
module.tectonic.random_id.admin_user_id: Refreshing state... (ID: WAflPu4wBzKAfb-ZcMrksA)
module.bootkube.tls_private_key.service-account: Refreshing state... (ID: 309c54c97a1175aea8ee103f9e36970b25ffecab)
module.bootkube.tls_locally_signed_cert.apiserver: Refreshing state... (ID: 257261979111112302356357538620711748845)
module.tectonic.tls_locally_signed_cert.identity-server: Refreshing state... (ID: 47680921552958291071870133850307687930)
module.tectonic.tls_locally_signed_cert.ingress: Refreshing state... (ID: 169072278900661547803222236154291192579)
module.tectonic.tls_locally_signed_cert.identity-client: Refreshing state... (ID: 61125144360372711194372730229589921213)
module.tectonic.random_id.console_secret: Refreshing state... (ID: c2zUibs8bxIkcf4_UQuKcA)
module.bootkube.tls_cert_request.apiserver: Refreshing state... (ID: 7e90de1c8f4826f06afd1e062cc9297e91bd81a6)
module.bootkube.tls_self_signed_cert.kube-ca: Refreshing state... (ID: 194380621406062159478355580167391944420)
module.bootkube.tls_private_key.apiserver: Refreshing state... (ID: 968316a0f73295c42f967a97006cf40cbe8c1512)
module.bootkube.tls_private_key.kube-ca: Refreshing state... (ID: 2dacf34e93af397e5214199ad0247b09a93140d7)
module.tectonic.tls_cert_request.identity-server: Refreshing state... (ID: 46349003398f328c75415de1b157b2483887c52d)
module.tectonic.tls_cert_request.ingress: Refreshing state... (ID: 66d6f456bf1f6582796020bdb45f03d18f1c5c5a)
module.tectonic.tls_cert_request.identity-client: Refreshing state... (ID: cdfc997d017002018bbc9a278f8fcd3179087495)
module.tectonic.tls_private_key.identity-server: Refreshing state... (ID: bfd2f98b4e5958152e253533f7a99c01222ba40f)
module.tectonic.tls_private_key.identity-client: Refreshing state... (ID: 963022979c273afd4e84cd30e9d3a4b948f7a157)
module.tectonic.tls_private_key.ingress: Refreshing state... (ID: abc9e9c91f2a7bf4b5887fe23a0522d56f19d879)
aws_s3_bucket_object.kubeconfig: Refreshing state... (ID: kubeconfig)
aws_route53_zone.tectonic-int: Refreshing state... (ID: Z2PYVCHSHEEY6R)
aws_s3_bucket_object.tectonic-assets: Refreshing state... (ID: assets.zip)
aws_s3_bucket.tectonic: Refreshing state... (ID: sym3tri-tf1.us-west-2.tectonic.dev.coreos.systems)
module.vpc.aws_route_table_association.worker_routing.2: Refreshing state... (ID: rtbassoc-c9f220b0)
module.vpc.aws_subnet.worker_subnet.2: Refreshing state... (ID: subnet-8823d7d3)
module.vpc.aws_security_group_rule.master_ingress_node_exporter: Refreshing state... (ID: sgrule-3424131731)
module.vpc.aws_security_group_rule.master_ingress_flannel_from_worker: Refreshing state... (ID: sgrule-2275253123)
module.vpc.aws_security_group_rule.master_ingress_kubelet_secure_from_worker: Refreshing state... (ID: sgrule-3652720449)
module.vpc.aws_security_group_rule.worker_ingress_kubelet_insecure_from_master: Refreshing state... (ID: sgrule-1164006892)
module.vpc.aws_security_group_rule.master_egress: Refreshing state... (ID: sgrule-3574500227)
module.vpc.aws_subnet.worker_subnet.0: Refreshing state... (ID: subnet-0b31aa42)
module.vpc.aws_security_group_rule.worker_ingress_services: Refreshing state... (ID: sgrule-804131367)
module.vpc.aws_security_group.etcd: Refreshing state... (ID: sg-c53b81be)
module.vpc.aws_route_table.private_routes.0: Refreshing state... (ID: rtb-fbab0f9d)
module.vpc.aws_security_group_rule.master_ingress_ssh: Refreshing state... (ID: sgrule-1643361510)
module.vpc.aws_security_group_rule.worker_ingress_services_from_console: Refreshing state... (ID: sgrule-2805068688)
module.vpc.aws_security_group_rule.master_ingress_http: Refreshing state... (ID: sgrule-3615091027)
module.vpc.aws_route.to_nat_gw.2: Refreshing state... (ID: r-rtb-15b014731080289494)
module.vpc.aws_security_group_rule.worker_ingress_flannel: Refreshing state... (ID: sgrule-212625757)
module.vpc.aws_security_group.api: Refreshing state... (ID: sg-763f850d)
module.vpc.aws_security_group_rule.master_ingress_kubelet_insecure_from_worker: Refreshing state... (ID: sgrule-994948218)
module.vpc.aws_eip.nat_eip.2: Refreshing state... (ID: eipalloc-d51176ef)
module.vpc.aws_security_group_rule.worker_ingress_heapster_from_master: Refreshing state... (ID: sgrule-2033912251)
module.vpc.aws_route_table_association.worker_routing.1: Refreshing state... (ID: rtbassoc-c1c91bb8)
module.vpc.aws_security_group_rule.master_ingress_https: Refreshing state... (ID: sgrule-3999523264)
module.vpc.aws_main_route_table_association.main_vpc_routes: Refreshing state... (ID: rtbassoc-22cc1e5b)
module.vpc.aws_security_group_rule.worker_egress: Refreshing state... (ID: sgrule-2721062876)
module.vpc.aws_route_table.private_routes.2: Refreshing state... (ID: rtb-15b01473)
module.vpc.aws_nat_gateway.nat_gw.1: Refreshing state... (ID: nat-054e217afc1449728)
module.vpc.aws_security_group_rule.master_ingress_services_from_console: Refreshing state... (ID: sgrule-3946382080)
module.vpc.aws_nat_gateway.nat_gw.0: Refreshing state... (ID: nat-045327159e80d10b3)
module.vpc.aws_security_group_rule.worker_ingress_node_exporter: Refreshing state... (ID: sgrule-1963850059)
module.vpc.aws_security_group_rule.master_ingress_icmp: Refreshing state... (ID: sgrule-422448188)
module.vpc.aws_security_group_rule.worker_ingress_flannel_from_master: Refreshing state... (ID: sgrule-1049131099)
module.vpc.aws_route.to_nat_gw.0: Refreshing state... (ID: r-rtb-fbab0f9d1080289494)
module.vpc.aws_security_group_rule.master_ingress_heapster: Refreshing state... (ID: sgrule-4060805989)
module.vpc.aws_security_group_rule.master_ingress_node_exporter_from_worker: Refreshing state... (ID: sgrule-4265347989)
module.vpc.aws_security_group_rule.master_ingress_heapster_from_worker: Refreshing state... (ID: sgrule-3224300131)
module.vpc.aws_subnet.master_subnet.1: Refreshing state... (ID: subnet-b691d1d1)
module.vpc.aws_nat_gateway.nat_gw.2: Refreshing state... (ID: nat-0a2d9813971e035a9)
module.vpc.aws_security_group_rule.master_ingress_flannel: Refreshing state... (ID: sgrule-3048835717)
module.vpc.aws_route_table.private_routes.1: Refreshing state... (ID: rtb-4bb5112d)
module.vpc.aws_route.igw_route: Refreshing state... (ID: r-rtb-b4b014d21080289494)
module.vpc.aws_security_group_rule.worker_ingress_ssh: Refreshing state... (ID: sgrule-2213567905)
module.vpc.aws_eip.nat_eip.0: Refreshing state... (ID: eipalloc-981077a2)
module.vpc.aws_security_group_rule.worker_ingress_icmp: Refreshing state... (ID: sgrule-2860660595)
module.vpc.aws_subnet.master_subnet.2: Refreshing state... (ID: subnet-362dd96d)
module.vpc.aws_security_group_rule.worker_ingress_http: Refreshing state... (ID: sgrule-897187860)
module.vpc.aws_security_group_rule.worker_ingress_kubelet_secure: Refreshing state... (ID: sgrule-2511398865)
module.vpc.aws_security_group_rule.worker_ingress_kubelet_secure_from_master: Refreshing state... (ID: sgrule-2811557591)
module.vpc.aws_subnet.master_subnet.0: Refreshing state... (ID: subnet-f33fa4ba)
module.vpc.aws_route_table_association.route_net.1: Refreshing state... (ID: rtbassoc-1bf32162)
module.vpc.aws_security_group_rule.worker_ingress_heapster: Refreshing state... (ID: sgrule-1260329149)
module.vpc.aws_security_group_rule.worker_ingress_https: Refreshing state... (ID: sgrule-2290974941)
module.vpc.aws_security_group_rule.worker_ingress_kubelet_insecure: Refreshing state... (ID: sgrule-2001028330)
module.vpc.aws_route_table_association.route_net.0: Refreshing state... (ID: rtbassoc-24cc1e5d)
module.vpc.aws_security_group_rule.master_ingress_kubelet_insecure: Refreshing state... (ID: sgrule-157918588)
module.vpc.aws_route.to_nat_gw.1: Refreshing state... (ID: r-rtb-4bb5112d1080289494)
module.vpc.aws_route_table_association.worker_routing.0: Refreshing state... (ID: rtbassoc-cec91bb7)
module.vpc.aws_eip.nat_eip.1: Refreshing state... (ID: eipalloc-bc117686)
module.vpc.aws_security_group_rule.master_ingress_kubelet_secure: Refreshing state... (ID: sgrule-3952870983)
module.vpc.aws_security_group_rule.master_ingress_services: Refreshing state... (ID: sgrule-1371730865)
module.vpc.aws_route_table_association.route_net.2: Refreshing state... (ID: rtbassoc-c8f220b1)
module.masters.aws_route53_record.ingress-private: Refreshing state... (ID: Z2PYVCHSHEEY6R_sym3tri-tf1.tectonic.dev.coreos.systems_A)
module.masters.aws_route53_record.api-internal: Refreshing state... (ID: Z2PYVCHSHEEY6R_sym3tri-tf1-api.tectonic.dev.coreos.systems_A)
module.masters.aws_route53_record.ingress-public: Refreshing state... (ID: Z1ILIMNSJGTMO2_sym3tri-tf1.tectonic.dev.coreos.systems_A)
module.vpc.aws_security_group_rule.worker_ingress_node_exporter_from_master: Refreshing state... (ID: sgrule-1193928781)
module.vpc.aws_subnet.worker_subnet.1: Refreshing state... (ID: subnet-1d92d27a)
module.workers.aws_iam_role_policy.worker_policy: Refreshing state... (ID: sym3tri-tf1-worker-role:sym3tri-tf1_worker_policy)
module.masters.aws_route53_record.api-external: Refreshing state... (ID: Z1ILIMNSJGTMO2_sym3tri-tf1-api.tectonic.dev.coreos.systems_A)
module.masters.aws_iam_role_policy.master_policy: Refreshing state... (ID: sym3tri-tf1-master-role:sym3tri-tf1_master_policy)
module.workers.aws_autoscaling_group.workers: Refreshing state... (ID: sym3tri-tf1-workers)
module.masters.aws_autoscaling_group.masters: Refreshing state... (ID: sym3tri-tf1-masters)
module.etcd.aws_instance.etcd_node.1: Refreshing state... (ID: i-0a86eaa1dc986ef02)
module.etcd.aws_instance.etcd_node.0: Refreshing state... (ID: i-0c46fa945c4a1f0b8)
module.etcd.aws_route53_record.etc_a_nodes.2: Refreshing state... (ID: Z2PYVCHSHEEY6R_sym3tri-tf1-etcd-2_A)
module.etcd.aws_route53_record.etc_a_nodes.0: Refreshing state... (ID: Z2PYVCHSHEEY6R_sym3tri-tf1-etcd-0_A)
module.etcd.aws_instance.etcd_node.2: Refreshing state... (ID: i-0b2ebcf4c9f045b8e)
module.etcd.aws_route53_record.etcd_srv_discover: Refreshing state... (ID: Z2PYVCHSHEEY6R__etcd-server._tcp_SRV)
module.etcd.aws_route53_record.etc_a_nodes.1: Refreshing state... (ID: Z2PYVCHSHEEY6R_sym3tri-tf1-etcd-1_A)
module.etcd.aws_route53_record.etcd_srv_client: Refreshing state... (ID: Z2PYVCHSHEEY6R__etcd-client._tcp_SRV)
module.vpc.aws_security_group.console: Refreshing state... (ID: sg-c73b81bc)
module.vpc.aws_internet_gateway.igw: Refreshing state... (ID: igw-6963880e)
module.vpc.aws_route_table.default: Refreshing state... (ID: rtb-b4b014d2)
module.vpc.aws_security_group.worker: Refreshing state... (ID: sg-113f856a)
module.vpc.aws_security_group.master: Refreshing state... (ID: sg-0c45ff77)
module.workers.aws_launch_configuration.worker_conf: Refreshing state... (ID: sym3tri-tf1-worker-0024bf3b878c8dd4244e507e40)
module.masters.aws_launch_configuration.master_conf: Refreshing state... (ID: sym3tri-tf1-master-0024bf3b878c8dd4244e507e3f)
module.masters.aws_elb.api-internal: Refreshing state... (ID: sym3tri-tf1-api-internal)
module.masters.aws_elb.console: Refreshing state... (ID: sym3tri-tf1-console)
module.masters.aws_elb.api-external: Refreshing state... (ID: sym3tri-tf1-api-external)
module.vpc.aws_vpc.new_vpc: Refreshing state... (ID: vpc-7b33071c)
module.workers.aws_iam_instance_profile.worker_profile: Refreshing state... (ID: sym3tri-tf1-worker-profile)
module.masters.aws_iam_instance_profile.master_profile: Refreshing state... (ID: sym3tri-tf1-master-profile)
module.masters.aws_iam_role.master_role: Refreshing state... (ID: sym3tri-tf1-master-role)
module.workers.aws_iam_role.worker_role: Refreshing state... (ID: sym3tri-tf1-worker-role)
Error refreshing state: 5 error(s) occurred:

* ignition_config.main: invalid file "01233d6410f7cc91e6bf728b29e1e72b53d134aafb5745d6bba154789d98041e", unknown file id
* ignition_config.etcd.1: invalid file "5c91abf93af056100841c461690fdb77dff529667c70a6f0237f33ded6fda3c9", unknown file id
* ignition_config.etcd.2: invalid systemd unit "297994451bbc66a0b227eea50017c4d05fde37b99130d1efc2f6b14b3bd2a283", unknown systemd unit id
* ignition_config.main: invalid file "01233d6410f7cc91e6bf728b29e1e72b53d134aafb5745d6bba154789d98041e", unknown file id
* ignition_config.etcd.0: invalid systemd unit "297994451bbc66a0b227eea50017c4d05fde37b99130d1efc2f6b14b3bd2a283", unknown systemd unit id
```

**Comment by Quentin-M** [comment]
This bug has always been here. This is the Ignition bug that sometimes happen on v0.8.8 and that ALWAYS happen on v0.9.2. It may happen at create OR at destroy. Your best bet is to remove references to these files in your tfstate, without forgetting to reduce the `#` of files (otherwise it'll crash, index out of array). Fixing that for good means we need to bump to the latest version - with the Ignition fix that @alexsomesan was talking about yesterday.

**Comment by sym3tri** [comment]
We will need to document an official workaround for any release we ship without that fix.

**Comment by alexsomesan** [comment]
@Quentin-M is right. You are hitting the bug that is preventing us from using 0.9.x
It's tracked here: https://github.com/hashicorp/terraform/issues/11518
In 0.8.x it should only manifest when resuming a partial `apply`
The only workaround is to destroy and re-apply from scratch if you hit it.
The fix just got merged! https://github.com/hashicorp/terraform/pull/13919

**Comment by alexsomesan** [comment]
Workaround to destory from CLI: pass `--refresh=false` to the destroy command:

`terraform destroy --refresh=false`

**Comment by Quentin-M** [comment]
Fixed by #219
